spinnaker/spinnaker_testing/gcp_quota_check.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
This strikes me as a weird place to write this test. The rest of this directory is full of classes that act as utilities for testing Spinnaker subsystems.